pflake8/__init__.py

```diff
@@ -51,16 +54,42 @@ class DivertingSafeConfigParser(ConfigParserTomlMixin, configparser.SafeConfigPa
     pass
 
 
-class ModifiedConfigFileFinder(flake8.options.config.ConfigFileFinder):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.project_filenames = self.project_filenames + ('pyproject.toml',)
-
-
 configparser.RawConfigParser = DivertingRawConfigParser
 configparser.ConfigParser = DivertingConfigParser
 configparser.SafeConfigParser = DivertingSafeConfigParser
-flake8.options.config.ConfigFileFinder = ModifiedConfigFileFinder
+
+
+class FixFilenames(ast.NodeTransformer):
+    tuple_of_interest = ("setup.cfg", "tox.ini", ".flake8")
+
+    def visit_Tuple(self, node: ast.Tuple) -> Any:
```

Comment:
Should this be named `visit_tuple` to follow PEP8 naming standards?

Reply:
It a typo.

Reply:
[`ast.NodeTransformer`](https://docs.python.org/3/library/ast.html#ast.NodeTransformer) actually calls the `visit_<NodeTypeName>` functions, so in this case it needs to be capitalized.